apps/web/src/app/(auth)/signin/page.tsx

```diff
@@ -0,0 +1,47 @@
+'use client'
+
+import { Button } from '@80000coding/ui'
+import { CategoryGithubRoundedIcon, StaticLogoSymbolIcon } from '@80000coding/web-icons'
+import { useRouter } from 'next/navigation'
+import { useCallback } from 'react'
+
+import TopbarWithXIcon from '@/components/common/TopbarWithXIcon'
+
+export default function SigninPage() {
+  const router = useRouter()
+
+  const handleClick = useCallback(() => {
+    /** TODO: Alert 생기면 교체 */
+    if (confirm('홈으로 이동하시겠습니까?')) router.push('/')
+  }, [router])
+
+  const handleGithubSignin = useCallback(() => {
+    /** TODO: github login 연결 */
+  }, [])
+
+  return (
+    <div className='flex h-full w-full flex-col'>
```

Comment:
NextUiProvider에 default height를 주는 것 보다 위 page.tsx의 최상위 div의 height를 100svh로 고정하는 방식은 어떨까요?
```suggestion
    <div className='flex h-[100svh] w-full flex-col'>
```

Reply:
@rkskekzzz  오 씻! svh 라는게 잇엇군요? 이건 귀하다
svh 대신 dvh 는 어떤가요??

Reply:
dvh 고고 하시죠!